### DISCUSSION TITLE
Query ~ contradiction in helptext wrt GUI->Expansions?

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1532] [Q&A] [answered]
Just now checking through the GUI, I notice Expansions --> helptext reads:

`uaenet.device is currently disabled.`

However, in current `master` the uaenet.device checkbox appears to be 'active' ...ie; users can enable this expansion....

Of course, this is all blocked by #1487 until resolved, and ideally `uaenet.device` should be disabled until that time ;)

As I'm unsure when v7.0 will be released, and if #1487 will be fixed by that time or not ....as such I feel it would be 'wrong' (read: disingenuous) to have this expansion enabled, when we know it's not yet working.... (and the contradiction in the helptext already/still exists)....

...yes, I'm being a pedant ~ just looking for clarification here?   8^)

TIA

**Comment by midwan** [comment, marked as answer]
I've updated the wiki.
When things improve, we'll update it again
